src/Markup/Avalonia.Markup.Xaml/XamlIl/CompilerExtensions/AvaloniaXamlIlCompiler.cs

```diff
@@ -40,13 +40,12 @@ void InsertBefore<T>(params IXamlAstTransformer[] t)
             
             
             // Targeted
-
             InsertBefore<PropertyReferenceResolver>(
                 new AvaloniaXamlIlTransformInstanceAttachedProperties(),
                 new AvaloniaXamlIlTransformSyntheticCompiledBindingMembers());
-            InsertAfter<PropertyReferenceResolver>(new AvaloniaXamlIlAvaloniaPropertyResolver());
-            
-
+            InsertAfter<PropertyReferenceResolver>(
+                new AvaloniaXamlIlAvaloniaPropertyResolver(),
+                new AvaloniaXamlIlResolveByNameMarkupExtensionReplacer());
```

Comment:
For style reasons, I’d suggest adding this to the `InsertBefore<ContentConvertTransformer>()` call. This pass is semantically converting the content of a property, not augmenting a property reference with additional information.

Reply:
done.